dscreator/datasets/timeseries/glomma.py

```diff
@@ -0,0 +1,96 @@
+            keywords=",".join(
+                [
+                    # GEMET & NORTHEMES
+                    "GCMDSK:EARTH SCIENCE > TERRESTRIAL HYDROSPHERE > SURFACE WATER > SURFACE WATER FEATURES > RIVERS/STREAMS",
+                    "GCMDLOC:CONTINENT > EUROPE > NORTHERN EUROPE > SCANDINAVIA > NORWAY",
+                    "GEMET:Hydrography",
+                    "NORTHEMES:Weather and climate",
+                ]
+            ),
+            keywords_vocabulary=",".join(
+                [
+                    "GCMDSK:GCMD Science Keywords:https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/sciencekeywords",
+                    "GCMDPLA:GCMD Platforms:https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/platforms",
+                    "GCMDLOC:GCMD Locations:https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/locations",
+                    "GEMET:INSPIRE Themes:http://inspire.ec.europa.eu/theme",
+                    "NORTHEMES:GeoNorge Themes:https://register.geonorge.no/metadata-kodelister/nasjonal-temainndeling",
+                ]
+            ),
+            iso_topic_category="inlandWaters",
+            featureType=ds.attrs["featureType"],
+            date_created=utils.iso_now(),
+            project="AquaINFRA",
+            time_coverage_start=utils.to_isoformat(ds.time.min().values),
+            time_coverage_end=utils.to_isoformat(ds.time.max().values),
+            geospatial_lat_min=float(ds.latitude.min()),
+            geospatial_lat_max=float(ds.latitude.max()),
+            geospatial_lon_min=float(ds.longitude.min()),
+            geospatial_lon_max=float(ds.longitude.max()),
+            spatial_representation="point",
+            collection="",
+        )
+
+    def variable_attributes(self, variable_name) -> dict:
+        """Match timeserie data to C&F
+
+        Match timeseries data to the climate and forecast convention based on the given variable code.
+        Standard names are found at http://vocab.nerc.ac.uk/collection/P07/current/
+        online unit list on https://ncics.org/portfolio/other-resources/udunits2/
+        """
+        match variable_name:
+            case "temp_water_avg":
+                return asdict(
+                    VariableAttrs(
+                        short_name="water_temperature", long_name="River Water Temperature", units="degree_Celsius"
+                    )
+                )
+            case "phvalue_avg":
+                return asdict(VariableAttrs(short_name="water_ph", long_name="River Water pH", units=""))
+            case "condvalue_avg":
+                return asdict(
+                    VariableAttrs(
+                        short_name="water_electrical_conductivity",
+                        long_name="River Water Conductivity",
+                        units="mS/m",
```

Comment:
Should be µS/cm, it's also wrong in grafana I noticed...

Reply:
and superset